configure

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env python
 
+import sys
+if sys.version_info >= (3, 0):
+  sys.stdout.write("Python 3.x is not compatible at this time\n")
```

Comment:
I'd skip "at this time" -- there's been no progress in supporting Python 3.x and I don't think we should hint that it might change.